src/XrdSciTokens/XrdSciTokensAccess.cc

```diff
@@ -1,6 +1,7 @@
 
 #include "XrdAcc/XrdAccAuthorize.hh"
 #include "XrdOuc/XrdOucEnv.hh"
+#include "XrdOuc/XrdOucStream.hh"
```

Comment:
If you use XrdOucGatherConf.hh then there is no need to include this file. See detailed notes in this at "Config()".

Reply:
Done in latest commits.